power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -108,6 +108,20 @@ template <symmetry_tag sym_type> struct PowerSensorCalcParam {
     RealValue<sym> q_variance{}; // variance (sigma^2) of the error range of the reactive power, in p.u.
 };
 
+// current sensor calculation parameters for state estimation
+// The value is the complex current
+//   * for appliances, it is always in injection direction
+//   * for branches, the direction is node -> branch
+template <symmetry_tag sym_type> struct CurrentSensorCalcParam {
+    using sym = sym_type;
+
+    static constexpr bool symmetric{is_symmetric_v<sym>};
+
+    ComplexValue<sym> value{};
+    RealValue<sym> i_variance{};       // variance (sigma^2) of the error range of the current, in p.u.
+    RealValue<sym> i_angle_variance{}; // variance (sigma^2) of the error range of the current angle, in p.u.
+};
```

Comment:
Since the `i_sigma` and `i_angle_sigma` are always single values (not three phase) as decided. The variance should also be always `double`. Just as voltage sensor.

Reply:
done